lib/rfxcom.js

```diff
@@ -66,50 +103,94 @@ util.inherits(RfxCom, EventEmitter);
 RfxCom.prototype.open = function() {
     var self = this;
 
-    if (typeof self.serialport === "undefined") {
-        self.serialport = new serialport.SerialPort(self.device, {
-            baudrate: 38400,
-            parser: self.rfxtrxParser()
+    if (typeof self.serialport === "undefined" || self.fakingSerialPort) {
```

Comment:
This doesn't look right, it shouldn't really care that it's a fake serial port?

I do wonder about simplifying it by requiring that you pass a SerialPort into the RfxCom constructor?